conda/core/subdir_data.py

```diff
@@ -240,26 +241,19 @@ def iter_records(self):
             self.load()
         return iter(self._package_records)
 
-    def _load_state(self) -> dict:
+    def _load_state(self):
```

Comment:
```suggestion
    def _load_state(self) -> CacheJsonState:
```
Does that help in any way?